tests/cloudpickle_test.py

```diff
@@ -2,6 +2,8 @@
 import pytest
 import pickle
 import sys
+import numpy as np
+import scipy.special
```

Comment:
I would take that approach too.
I don't know though if can we make these test dependencies optional?